.travis.yml

```diff
@@ -3,23 +3,19 @@ sudo: required
 notifications:
   email: false
 otp_release:
-  - 21.0.1
-  - 20.3
-  - 19.3
-  - 19.2
-  - 19.1
   - 19.0
-  - 18.3
-  - 18.2.1
-  - 18.1
-  - 17.5
-  - 17.4
-  - 17.3
-  - 17.1
-  - 17.0
-  - R16B03-1
-  - R16B03
-  - R16B02
-  - R16B01
+  - 19.3
```

Comment:
Why not 19.1, 19.2 and 20.2?

Reply:
They're not available on Travis  - they stopped being available a few months ago. (Perhaps the situation has changed in the mean time.)